### DISCUSSION TITLE
2.3.10 Yara Updates

### DISCUSSION THREAD

**Discussion by bayden10** [Security-Onion-Solutions/securityonion#2342] [Unsupported Versions] [answered]
Hi
Was not sure what the heartbeat is for Yara so I ran manually.. What would that be?

```
#so-yara-update
Cloning into '/tmp/signature-base'...
remote: Enumerating objects: 32, done.
remote: Counting objects: 100% (32/32), done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 5714 (delta 16), reused 20 (delta 8), pack-reused 5682
Receiving objects: 100% (5714/5714), 33.74 MiB | 15.79 MiB/s, done.
Resolving deltas: 100% (3811/3811), done.
Analyzing rules from /tmp/signature-base...
Adding new rule: apt_solarwinds_sunburst.yar...
Done!
```

**Comment by weslambert** [comment, marked as answer]
> Hi
> Was not sure what the heartbeat is for Yara so I ran manually.. What would that be?
> 
> ```
> #so-yara-update
> Cloning into '/tmp/signature-base'...
> remote: Enumerating objects: 32, done.
> remote: Counting objects: 100% (32/32), done.
> remote: Compressing objects: 100% (24/24), done.
> remote: Total 5714 (delta 16), reused 20 (delta 8), pack-reused 5682
> Receiving objects: 100% (5714/5714), 33.74 MiB | 15.79 MiB/s, done.
> Resolving deltas: 100% (3811/3811), done.
> Analyzing rules from /tmp/signature-base...
> Adding new rule: apt_solarwinds_sunburst.yar...
> Done!
> ```

`so-yara-update` currently runs daily at 7:01 UTC.
